src/main/java/it/gov/pagopa/pu/organization/service/organization/OrganizationSilService.java

```diff
@@ -0,0 +1,39 @@
+package it.gov.pagopa.pu.organization.service.organization;
+
+import it.gov.pagopa.pu.organization.dto.generated.OrgSilServiceDTO;
+import it.gov.pagopa.pu.organization.exception.custom.OrgSilServiceNotFoundException;
+import it.gov.pagopa.pu.organization.mapper.OrgSilServiceMapper;
+import it.gov.pagopa.pu.organization.model.OrgSilService;
+import it.gov.pagopa.pu.organization.repository.OrgSilServiceRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor
+public class OrganizationSilService {
```

Comment:
Even if ugly... you cannot name it this way... what should be the difference between the entity name OrgSilService and the Service?
```suggestion
public class OrganizationSilServiceService {
```

Reply:
this has not been resolved